src/www/ui/api/Controllers/LicenseController.php

```diff
@@ -473,7 +483,8 @@ public function getAllAdminAcknowledgements($request, $response, $args)
     foreach ($res as $key => $ack) {
       $res[$key]['id'] = intval($ack['la_pk']);
       unset($res[$key]['la_pk']);
-      $res[$key]['is_enabled'] = $ack['is_enabled'] == "t";
+      $res[$key]['isEnabled'] = $ack['is_enabled'] == "t";
+      unset($res[$key]['is_enabled']);
```

Comment:
The field `is_enabled` is still required for v1 response, right?

Reply:
Yes, I will create a new model for this response and address it there.